app/models/pusher.rb

```diff
@@ -110,7 +110,7 @@ def update_remote_bundler_api(to = RestClient)
     }.to_json
 
     begin
-      timeout(5) do
+      ::Timeout.timeout(5) do
         to.post @bundler_api_url,
           json,
           :timeout        => 5,
```

Comment:
> can we set a timeout on the RestClient instance instead?

Doesn't the above line already does that?
And to answer my question... [[Restclient readme](https://github.com/rest-client/rest-client#passing-advanced-options)]
> The top level helper methods like RestClient.get accept a headers hash as their last argument and don't allow passing more complex options

This should be to.`RestClient::Request.execute(method: :post, ....) I hope I got this right.